docs/pages/components/tables.vue

```diff
@@ -404,33 +420,33 @@ You can then use the set sortedColumns prop on the KtTable to update the table u
 </KtTable>
 
 ```html
-<KtTable :rows="rows" :columns="columns">
```

Comment:
Does this example really make sense without `columns`?

Reply:
no, u r right